vm/float.go

```diff
@@ -444,7 +445,11 @@ func (f *FloatObject) lessThan(arg Object) bool {
 // ToString returns the object's value as the string format, in non
 // exponential format (straight number, without exponent `E<exp>`).
 func (f *FloatObject) ToString() string {
-	return strconv.FormatFloat(f.value, 'f', -1, 64) // fmt.Sprintf("%f", f.value)
+	s := strconv.FormatFloat(f.value, 'f', -1, 64)
+	if strings.IndexAny(s, ".") == -1 {
```

Comment:
Can you add a comment to explain this case?

Reply:
Added a comment, and revised the line to be more clear.